bin/install-docker.js

```diff
@@ -0,0 +1,93 @@
+const path = require( 'path' );
+const Docker = require('dockerode');
+const compose = require('docker-compose');
+const docker = new Docker();
+const yml = path.join(__dirname) + '/../';
+
+class DockerInstaller {
+	async init() {
+		const result = await this.dockerInfo();
+		return result;
+	}
+
+	/**
+	 * Check that Docker is installed and running.
+	 */
+	async dockerInfo() {
+		return new Promise( ( res ) => {
+			docker.info()
+				.then( async () => {
+					const resp = await this.composeStop();
+					return res( resp );
+				} )
+				.catch( ( err ) => { this.onDockerError( err ) } );
+		} );
+	}
+
+	/**
+	 * Stop existing containers.
+	 * This stops all running containers, including those on port 80.
+	 */
+	async composeStop() {
```

Comment:
In our shell script we had a section where we stopped all containers on port 80, not just the current defined in compose. Does this do that?

Reply:
This should shut down every running container. I tested it by starting several containers and every single one gut shut down.

The main problem to implement the extra check for port 80 is `curl` W hoch is no default command on windows. Might need to research a package that could wrap curl or similiar if the check is neccessary.